neo4j/datadog_checks/neo4j/data/conf.yaml.example

```diff
@@ -2,361 +2,60 @@
-
-    ## @param tls_cert - string - optional
-    ## The path to a single file in PEM format containing a certificate as well as any
-    ## number of CA certificates needed to establish the certificate's authenticity for
-    ## use when connecting to services. It may also contain an unencrypted private key to use.
-    #
-    # tls_cert: <CERT_PATH>
-
-    ## @param tls_private_key - string - optional
-    ## The unencrypted private key to use for `tls_cert` when connecting to services. This is
-    ## required if `tls_cert` is set and it does not already contain a private key.
-    #
-    # tls_private_key: <PRIVATE_KEY_PATH>
-
-    ## @param tls_ca_cert - string - optional
-    ## The path to a file of concatenated CA certificates in PEM format or a directory
-    ## containing several CA certificates in PEM format. If a directory, the directory
-    ## must have been processed using the c_rehash utility supplied with OpenSSL. See:
-    ## https://www.openssl.org/docs/manmaster/man3/SSL_CTX_load_verify_locations.html
-    #
-    # tls_ca_cert: <CA_CERT_PATH>
-
-    ## @param headers - mapping - optional
-    ## The headers parameter allows you to send specific headers with every request.
-    ## You can use it for explicitly specifying the host header or adding headers for
-    ## authorization purposes.
-    ##
-    ## This overrides any default headers.
-    #
-    # headers:
-    #   Host: <ALTERNATIVE_HOSTNAME>
-    #   X-Auth-Token: <AUTH_TOKEN>
-
-    ## @param extra_headers - mapping - optional
-    ## Additional headers to send with every request.
-    #
-    # extra_headers:
-    #   Host: <ALTERNATIVE_HOSTNAME>
-    #   X-Auth-Token: <AUTH_TOKEN>
-
-    ## @param timeout - number - optional - default: 10
-    ## The timeout for accessing services.
-    ##
-    ## This overrides the `timeout` setting in `init_config`.
-    #
-    # timeout: 10
-
-    ## @param connect_timeout - number - optional
-    ## The connect timeout for accessing services. Defaults to `timeout`.
-    #
-    # connect_timeout: <CONNECT_TIMEOUT>
-
-    ## @param read_timeout - number - optional
-    ## The read timeout for accessing services. Defaults to `timeout`.
-    #
-    # read_timeout: <READ_TIMEOUT>
-
-    ## @param log_requests - boolean - optional - default: false
-    ## Whether or not to debug log the HTTP(S) requests made, including the method and URL.
-    #
-    # log_requests: false
-
-    ## @param persist_connections - boolean - optional - default: false
-    ## Whether or not to persist cookies and use connection pooling for increased performance.
-    #
-    # persist_connections: false
+    # exclude_labels:
+    #  - <TAG_1>
+    #  - <TAG_2>
 
     ## @param tags - list of strings - optional
     ## A list of tags to attach to every metric and service check emitted by this instance.
     ##
-    ## Learn more about tagging at https://docs.datadoghq.com/tagging
+    ## Learn more about tagging at [Datadog Getting Started][1]
```

Comment:
Seems like CI is failing because it doesn't like use of `[1]` and `[2]` for links. I think keeping it the old way is fine.

Reply:
Went back to old format. Let me know if that works.

Reply:
I don't think you pushed your changes yet.